### DISCUSSION TITLE
Is TF-IDF normalized count used in differential accessibility analysis

### DISCUSSION THREAD

**Discussion by qingwang1** [stuart-lab/signac#776] [Q&A] [answered]
Hi Signac team,

I am running differential accessibility analysis and it seems to me that by default in FindMarkers(.....,test.use = 'LR',
  latent.vars = 'peak_region_fragments'), it's using the 'data' slot which is the log1p of TF-IDF count, right? So if I'd like to do a sanity check on the DA peak count between the conditions, should I use the mean raw count using AverageExpression(slot='count')(as I saw in some other post in the discussion forum), or use the AverageExpression(slot='data')? Many thanks!

**Comment by timoast** [comment, marked as answer]
By default the TF-IDF matrix is used for differential accessibility. You can use raw counts by setting `slot="counts"` in `FindMarkers()`. Since comparisons here are done across cells (within a peak), the IDF part is a constant. Using TF-IDF therefore is normalizing for differences in total counts across cells. In practice, I'm not sure how much benefit there is to using TF-IDF values, especially if you include sequencing depth as a latent variable in the test.
